presto-docs/src/main/sphinx/release/release-0.206.rst

```diff
@@ -0,0 +1,52 @@
+=============
+Release 0.206
+=============
+
+General Changes
+---------------
+
+* Fix query execution failure for certain queries containing a join followed by an aggregation
+  when ``dictionary_aggregation`` is enabled.
+* Fix query plan for aggregations where ``GROUP BY`` is present, but cardinality of the
+  grouping columns is 1, e.g., ``SELECT c1, sum(c2) FROM t WHERE c1 = 'foo' GROUP BY c1``.
+* Fix high memory pressure on the coordinator during the execution of queries
+  using bucketed execution.
+* Add new semantics for date/time types. The new semantics conforms to the SQL spec.
```

Comment:
This is a big change. Let's move this to a separate separate section and explain the change in detail.

Reply:
It is. On the other hand, it's also not very useful at this stage, as no connector has been updated yet. Maybe we can trumpet more about this when end-users can actually benefit?